ui/intermittents-view/constants.js

```diff
@@ -0,0 +1,37 @@
+import * as d3 from "d3";
+
+// data is passed as a prop and target is updated via a ref in graph.jsx
+export const graphOneSpecs = {
+  title: "Orange Count per Push",
```

Comment:
Could you switch the quotes to single for JS throughout? We haven't yet been able to enable the eslint rule (since have to fix the old instances first), but for new code we're following the AirBnB style guide, which says single quotes for JS and double for JSX (latter since HTML attributes use double):
https://github.com/airbnb/javascript/tree/master/react#quotes